js/src/personal/tony/Tony.module.css

```diff
@@ -1,15 +1,29 @@
 .title {
-  color: light-dark(var(--mantine-color-black), var(--mantine-color-white));
-  font-size: rem(100px);
-  font-weight: 900;
-  letter-spacing: rem(-2px);
-
+  font-size: rem(60px);
```

Comment:
Your commit message should be in the format:
```
Tony: Main Summary

Longer description and more minor changes

L=ENG-xxx
```

Also the first line should average 50 characters and not be longer than 72 characters.
No line should be longer than 72 chars unless you're like pasting a URL.